### PR TITLE
Deals with issue #63

### PR DESCRIPTION
**Pull request by rohe**
Removed by_schema() usage. Its usage in the context was inappropriate.

### CODE REVIEW

File: src/oidcendpoint/id_token.py [hunk 1]
```diff
@@ -189,16 +191,16 @@ def payload(
...
+            self,
+            session_info,
+            client_id,
+            code=None,
+            access_token=None,
+            user_info=None,
+            sign=True,
+            encrypt=False,
+            lifetime=None,
+            extra_claims=None,
```
Comment (nsklikas) [inline src/oidcendpoint/id_token.py]: Fix format

File: src/oidcendpoint/id_token.py [hunk 2]
```diff
@@ -119,16 +121,16 @@ def __init__(self, endpoint_context, **kwargs):
...
+            self,
+            session,
+            acr="",
+            alg="RS256",
+            code=None,
+            access_token=None,
+            user_info=None,
+            auth_time=0,
+            lifetime=None,
+            extra_claims=None,
```
Comment (nsklikas) [inline src/oidcendpoint/id_token.py]: Fix format

File: src/oidcendpoint/id_token.py [hunk 3]
```diff
@@ -50,7 +52,7 @@ def include_session_id(endpoint_context, client_id, where):
 
 
 def get_sign_and_encrypt_algorithms(
-    endpoint_context, client_info, payload_type, sign=False, encrypt=False
+        endpoint_context, client_info, payload_type, sign=False, encrypt=False
```
Comment (nsklikas) [inline src/oidcendpoint/id_token.py]: Fix format

**Review by nsklikas — COMMENTED** [review]
Other than the formatting errors it LGTM.

File: src/oidcendpoint/id_token.py (hunk 3 above)
Comment (rohe) [reply src/oidcendpoint/id_token.py]: That's what PyCharm does when it reformats code. Specifically for method/function arguments.

File: src/oidcendpoint/id_token.py (hunk 1 above)
Comment (rohe) [reply src/oidcendpoint/id_token.py]: Used black to reformat.

File: src/oidcendpoint/id_token.py (hunk 2 above)
Comment (rohe) [reply src/oidcendpoint/id_token.py]: Used black to fix format.

File: src/oidcendpoint/id_token.py (hunk 3 above)
Comment (rohe) [reply src/oidcendpoint/id_token.py]: Used black to fix format.

**Pull request merged**